From 5977fcd8a729d3a40c8373e62702df45fb721bd0 Mon Sep 17 00:00:00 2001
From: Selim Cinek <cinek@google.com>
Date: Tue, 20 Feb 2018 09:54:20 -0800
Subject: [PATCH 140/306] Fixed Security Exception issue when inflating
 notification

This is a temporary fix until we can resolve why the permissing
isn't granted properly.

Bug: 73640774
Test: receive mms, reply from hun
Change-Id: I77d909762d90c84b95f32c4c20c495ba46b5a176
(cherry picked from commit e268fda3c55598fbf614c55b8de10720ba7ac1f0)
---
 .../java/com/android/internal/widget/MessagingImageMessage.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/com/android/internal/widget/MessagingImageMessage.java b/core/java/com/android/internal/widget/MessagingImageMessage.java
index 961f90a3c11..9db74e82e38 100644
--- a/core/java/com/android/internal/widget/MessagingImageMessage.java
+++ b/core/java/com/android/internal/widget/MessagingImageMessage.java
@@ -97,7 +97,7 @@ public class MessagingImageMessage extends ImageView implements MessagingMessage
         Drawable drawable;
         try {
             drawable = LocalImageResolver.resolveImage(message.getDataUri(), getContext());
-        } catch (IOException e) {
+        } catch (IOException | SecurityException e) {
             e.printStackTrace();
             return false;
         }
-- 
2.17.1

